main.cc

```diff
@@ -121,9 +123,12 @@ class Matrix_t : public repl_new
     // localalloc a single row
     void allocateRow(Index_t i)
     {
-        for (Index_t j = 0; j < nrows_per_nodelet_; ++j)
+        for (Index_t row_idx= 0; row_idx < nrows_; ++row_idx)
         {
-            new(rows_[i] + j) Row_t();
+ 		size_t nid(n_map(row_idx));
```

Comment:
Please maintain original indentation.
Please use the type that the function returns.

Reply:
fixed in the next commit